diff --git a/node_modules/vite-plugin-solid/dist/esm/index.mjs b/node_modules/vite-plugin-solid/dist/esm/index.mjs
index 735444a..bb94e84 100644
--- a/node_modules/vite-plugin-solid/dist/esm/index.mjs
+++ b/node_modules/vite-plugin-solid/dist/esm/index.mjs
@@ -130,7 +130,7 @@ function solidPlugin(options = {}) {
         if (config.consumer === 'client' || name === 'client' || opts.isSsrTargetWebworker) {
           config.resolve.conditions = [...defaultClientConditions];
         } else {
-          config.resolve.conditions = [...defaultServerConditions];
+          config.resolve.conditions = [...(defaultServerConditions || [])];
         }
       }
       config.resolve.conditions = ['solid', ...(replaceDev ? ['development'] : []), ...(isTestMode && !opts.isSsrTargetWebworker ? ['browser'] : []), ...config.resolve.conditions];
